src/__tests__/react-router.js

```diff
@@ -47,32 +47,34 @@ const App = () => (
 
 // this is a handy function that I would utilize for any component
 // that relies on the router being in context
-function renderWithRouter(ui, {route = '/'} = {}) {
+const renderWithRouter = (ui, {route = '/'} = {}) => {
```

Comment:
Haha, I don't like this change (https://kentcdodds.com/blog/function-forms), but it's no big deal.

Why don't we call this function `render` and alias the `render` that we import from `@testing-library/react` to `rtlRender`.